### PR TITLE
Add libdmraid package

### PR DESCRIPTION
**Pull request by Zopolis4**
## Description
Adds the libdmraid package (part of the same larger package as #6766).

## Additional information
Updates lvm2 to compile the deventd daemon as that's a dependency of libdmraid, and bumps the version of lvm2 to avoid conflicts. Also updates the .gitignore to prevent Visual Studio or build files to accidentally being uploaded.

Works properly:
- [x] armv7l

### Run the following to get this pull request's changes locally for testing.
```
CREW_TESTING_REPO=https://github.com/Zopolis4/chromebrew.git CREW_TESTING_BRANCH=dmraid CREW_TESTING=1 crew update
```
---

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,4 @@ release/
 *.sha256
 *.xz
 *.zip
+/.vs
```
Comment (uberhacker) [inline .gitignore:7]: A `.gitignore` is actually not a bad idea.  You could add `*.gz` too.

Comment (Zopolis4) [reply .gitignore:7]: *.gz is already present in the .gitignore, in line 3

Comment (uberhacker) [reply .gitignore:7]: Sorry, `*.tgz`.

**Comment by uberhacker** [comment]
I tried building libdmraid on x86_64 and receive the following error:
```
checking libdevmapper-event.h usability... no
checking libdevmapper-event.h presence... no
checking for libdevmapper-event.h... no
configure: error: Missing headers device-mapper headers
/usr/local/lib/crew/lib/package.rb:166:in `system': Command failed with exit 1: bash
libdmraid failed to build: `CFLAGS="-O2 -pipe -flto -ffat-lto-objects -fPIC -fuse-ld=gold" CXXFLAGS="-O2 -pipe -flto -ffat-lto-objects -fPIC -fuse-ld=gold" FCFLAGS="-O2 -pipe -flto -ffat-lto-objects -fPIC -fuse-ld=gold" FFLAGS="-O2 -pipe -flto -ffat-lto-objects -fPIC -fuse-ld=gold" LDFLAGS="-flto" bash -c ./1.0.0.rc16-3/dmraid/configure` exited with 1
```

**Comment by Zopolis4** [comment]
You're gonna need to update lvm2

**Comment by satmandu** [comment]
Wait, is dmraid support useful on any shipped ChromeOS hardware ever?

Isn't mdadm the software which has superseded it?

Dmraid is from ~ 2006...

**Comment by satmandu** [comment]
Doing a rebuild of lvm2 now with the options in this PR added. Will add the builds to this PR after I am done.

**Comment by satmandu** [comment]
Binaries added for libdmraid. This should be good to go.

**Comment by Zopolis4** [comment]
> Wait, is dmraid support useful on any shipped ChromeOS hardware ever?
> 
> Isn't mdadm the software which has superseded it?
> 
> Dmraid is from ~ 2006...

Libblockdev still requires it to compile, so maybe it's just covering all bases.

**Pull request merged**